src/using-workflow/using-docker-images.md

```diff
@@ -58,12 +58,30 @@ Use `deis scale cmd=3` to increase `cmd` processes to 3, for example. Scaling a
 process type directly changes the number of [Containers][container]
 running that process.
 
-!!! attention
-    Support for Docker registry authentication is coming soon.
+## Private Registry
 
+To deploy Docker images from a private registry (either from a self-hosted registry or one of the public ones),
+take the following steps:
```

Comment:
Suggested change:

"To deploy Docker images from a private registry or even from a private repository within a public registry, take the following steps:"